README.md

```diff
@@ -13,7 +13,7 @@ This script is meant for those who are intimidated by installing DWM by themselv
 * The script has no error reporting. So if it errors out, the only notice you'll get is in the script output. I'm aware that this is not best practice, but I'm still learning bash.
 * If you do not use the standard Download folder stucture (i.e $HOME/Downloads) you will need to edit the script and change the download of my scripts folder to whereever you please. 
 * After install you'll need to run crontab -e and add in the crontab below. If you do not, several things will not work.
-* This may or may not work on vanilla Arch Linux, depending on what repos you have enabled in your pacman.conf file. If you get DWM and everything installed, but can't open a terminal, make sure you manually install alacritty, sxhkd, and rofi. This is something that I'll work on in the future. 
+* This may or may not work on vanilla Arch Linux, depending on what repos you have enabled in your pacman.conf file.
```

Comment:
It seems like dependencies are now installed [here](https://github.com/mtwb47/dwmricer/blob/master/dwminstall.sh#L6:L13)